docs/components/table-view.md

```diff
@@ -510,6 +510,48 @@ interface TablePaginationAttributes {
 
 A boolean to hide pagination element (defaults to `false`).
 
+### expandableRows
```

Comment:
What if we only want to allow a subset of rows to be expandable?

Reply:
Example, a list of control planes. Some are composite and have expandable rows, others are not composite.

Reply:
Oh.. you're right. I guess this prop needs to be a function that returns a boolean